requirements.txt

```diff
@@ -1,4 +1,4 @@
-appdirs>=1.4.4
+platformdirs
```

Comment:
Can you add the latest version `platformdirs>=3.6.0`?

Reply:
OK.  Done.